apps/condo/domains/acquiring/schema/GeneratePaymentLinkService.test.js

```diff
@@ -56,17 +56,15 @@ const receiptData = () => ({
 describe('GeneratePaymentLinkService', () => {
     let adminClient
     let acquiringIntegrationContext
-    let dummyO10n, dummyIntegration, dummyInvoiceContext
+    let dummyO10n
 
     beforeAll(async () => {
         adminClient = await makeLoggedInAdminClient()
 
         ;[dummyO10n] = await createTestOrganization(adminClient)
         await createTestBillingIntegration(adminClient)
         const [acquiringIntegration] = await createTestAcquiringIntegration(adminClient)
-        ;[acquiringIntegrationContext] = await createTestAcquiringIntegrationContext(adminClient, dummyO10n, acquiringIntegration)
-        ;[dummyIntegration] = await createTestAcquiringIntegration(adminClient)
-        ;[dummyInvoiceContext] = await createTestInvoiceContext(adminClient, dummyO10n, dummyIntegration, { status: INVOICE_CONTEXT_STATUS_FINISHED })
+        ;[acquiringIntegrationContext] = await createTestAcquiringIntegrationContext(adminClient, dummyO10n, acquiringIntegration, { invoiceStatus: CONTEXT_FINISHED_STATUS })
```

Comment:
wtf?
where did you get a syntax like this??)

Reply:
The main reason is to prevent the creating of extra variable to pass value from create to global variable.

Reply:
If your lines starts with `;` then you and your linter are probably doing something wrong :)))
`;` must be at the end of previous line, in the beginning of the new one :)

Reply:
fixed